packages/console/src/pages/Applications/index.tsx

```diff
@@ -18,9 +19,9 @@ import CreateForm from './components/CreateForm';
 import * as styles from './index.module.scss';
 
 const Applications = () => {
-  const [isCreateFormOpen, setIsCreateFormOpen] = useState(false);
+  const [isCreateFormOpen, setIsCreateFormOpen] = useState(true);
```

Comment:
is it intended to keep the createForm dialog open by default?

Reply:
no it's a mistake. just for debug